rust/gitxetcore/src/git_integration/git_repo.rs

```diff
@@ -513,6 +554,24 @@ impl GitRepo {
         Ok(())
     }
 
+    /// Set up a repo when the filter is run but no notes are present.
+    /// This can happen on certain
+    ///
+    /// May be run multiple times without changing the current configuration.
+    pub fn install_gitxet_from_filter_process(&self) -> Result<()> {
+        // Initialize a local version of the
+        info!(
+            "Configure Merkle DB and repo salt associated with repo {:?}",
+            &self.repo_dir
+        );
+
+        self.set_repo_mdb_to_v2_from_uninitialized()?;
+
+        // self.set_repo_salt()?;
```

Comment:
Why do we skip setting repo salt?

Reply:
Also delete the comment if it's truly not needed. The behavior regarding the salt should be write it if it's not found correct?

Reply:
This here was left over from debugging the previous command, where it was actually running the filter before the notes were fetched and thus causing the repo salts to end up conflicting.  Now we could call this, as the salt has been fetched, but in the github case it's been set remotely.